### PR TITLE
Converted RengoManagementPane to a function component

### PR DESCRIPTION
**Pull request by GreenAsJade**
Fixes future ugliness when adding rdh context

## Proposed Changes

Convert RengoManagementPane to be a function component.

### CODE REVIEW

**Comment by GreenAsJade** [comment]
Note: adding rdh to a class based component is not _that_ bad, I just prefer to have one way of doing it if possible!

File: src/components/RengoManagementPane/RengoManagementPane.tsx [hunk 1]
```diff
@@ -1,18 +1,18 @@
 /*
  * Copyright (C) 2012-2017  Online-Go.com
  *
- * This program is free software: you can redistribute it and/or modify
```
Comment (anoek) [inline src/components/RengoManagementPane/RengoManagementPane.tsx:4]: I reckon we should retain the This's in the legal preamble :)

Comment (GreenAsJade) [reply src/components/RengoManagementPane/RengoManagementPane.tsx:4]: Whoa, how on earth did I delete that line!!?

Comment (GreenAsJade) [reply src/components/RengoManagementPane/RengoManagementPane.tsx:4]: Oh LOL ... I deleted all the "this" :D

Comment (BHydden) [reply src/components/RengoManagementPane/RengoManagementPane.tsx:4]: Good old replace all haha wHaT cOuLd Go WrOnG 🤪🫠😂

**Comment by anoek** [comment]
Thanks, even though you now spell it as `THis` for one case :rofl:

**Pull request merged**